src/models/ITestResultPayload.ts

```diff
@@ -36,4 +36,11 @@ export interface ITestResultPayload {
   make?: string;
   model?: string;
   bodyType?: BodyTypeModel;
+  centralDocs?: CentralDocs;
```

Comment:
Not sure if I have missed something or a conversation, but is there a need to have this at the top level of the Test Result? I thought it was only going to be an optional in each test type?

Reply:
It is required for only the test types object, i've removed it now

Reply:
That was my understanding, unless something has changed.